src/components/Home/Button.jsx

```diff
@@ -1,9 +1,38 @@
-import React from 'react'
+/* eslint-disable react/prop-types */
```

Comment:
delete all the button code and reset it to default component

Reply:
ok done